flang/lib/Evaluate/intrinsics.cpp

```diff
@@ -1939,7 +1945,7 @@ std::optional<SpecificCall> IntrinsicInterface::Match(
   int elementalRank{0};
   for (std::size_t j{0}; j < dummies; ++j) {
     const IntrinsicDummyArgument &d{dummy[std::min(j, dummyArgPatterns - 1)]};
-    if (const ActualArgument *arg{actualForDummy[j]}) {
+    if (const ActualArgument * arg{actualForDummy[j]}) {
```

Comment:
These seem to be unrelated formatting changes.  Would you mind taking them back to keep the PR focused on the new feature?  (Same for the other formatting-only changes.)

Reply:
Thank you for your feedback. I'll correct this.